src/assets/AssetsDetectionController.ts

```diff
@@ -355,9 +355,10 @@ export class AssetsDetectionController extends BaseController<
     if (!this.isMainnet()) {
       return;
     }
-    const { selectedAddress } = this.config;
+    const requestedSelectedAddress = this.config.selectedAddress;
```

Comment:
any particular reason for the rename? seems `selectedAddress` would still be fine no?

Reply:
hm not really, just to be clear that it was the address from where it was being called

Reply:
I can go back to selectedAddress if you want 👌

Reply:
nah it's fine.

> just to be clear that it was the address from where it was being called

I think that's plenty reason enough to rename. feel free to merge.